### PR TITLE
chore: add SelectorBar to ui collection

### PR DESCRIPTION
**Pull request by Mohammer5**
Question: @varl how do I make this PR make the workflows publish a new version? All the commits are chores

### CODE REVIEW

**Comment by varl** [comment]
There has to be a fix or feat commit.

File: storybook/config/preview.js [hunk 1]
```diff
@@ -64,16 +64,18 @@ export const parameters = {
...
-            <>
-                <Title />
-                <Subtitle />
-                <Description />
-                <Primary />
-                <ArgsTable story={PRIMARY_STORY} />
-                <Stories title="Examples" />
-            </>
-        ),
+        page: function Page() {
```
Comment (varl) [inline storybook/config/preview.js:67]: Out of curiousity, why the change to a named function ?

**Review by varl — APPROVED** [review]
We could consider it a bug that we forgot to export it in the initial feat release (7.10.0) and release this as a patch.

Then they would get this fix even if their version range on UI is strict.

File: storybook/config/preview.js (hunk 1 above)
Comment (Mohammer5) [reply storybook/config/preview.js:67]: I got a prettier error.. Dunno why

Comment (varl) [reply storybook/config/preview.js:67]: Huh.

**Comment by Mohammer5** [comment]
> We could consider it a bug that we forgot to export it in the initial feat release (7.10.0) and release this as a patch.

I reworded the second commit from `feat` to `bug`

**Review by Birkbjo — APPROVED** [review]
